src/components/data/common/Pagination/PaginationInfo.tsx

```diff
@@ -1,16 +1,27 @@
-import React from 'react';
+import React, { ComponentProps, ReactElement } from 'react';
 import PropTypes from 'prop-types';
-import _ from 'lodash';
 
 import Form from 'components/form/Form';
+import { DropdownItemProps } from 'semantic-ui-react';
 
-/**
- * @return {null}
- */
+interface PaginationInfoProps {
+    pageSize: number;
+    onPageSizeChange: (pageSize: string) => void;
+    totalSize: number;
+    currentPage: number;
+    sizeMultiplier: number;
```

Comment:
As they are optional, shouldn't we define them this way:
```suggestion
    currentPage?: number;
    sizeMultiplier?: number;
```
?

Reply:
That's the magic of TS and `defaultProps`. The short answer is: they should be required and TypeScript does some magic because the properties are optional when you are trying to use the component, as they appear in `defaultProps`

Proof:


![image](https://user-images.githubusercontent.com/889383/127126495-10527b59-092c-45de-85c5-56b9c09e55c0.png)


https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/default_props/ for some more information on the topic (but it does not describe it too well)

Reply:
OK, clear 👍🏻 Thanks for explanation.